frontend/templates/product-list/MetaTags.tsx

```diff
@@ -11,92 +11,141 @@ import {
 import { jsonLdScriptProps } from 'react-schemaorg';
 import { BreadcrumbList as SchemaBreadcrumbList } from 'schema-dts';
 import { useDevicePartsItemType } from './hooks/useDevicePartsItemType';
+import { useItemTypeOverrides } from './hooks/useItemTypeOverrides';
 import { useProductListBreadcrumbs } from './hooks/useProductListBreadcrumbs';
 
 export interface MetaTagsProps {
    productList: ProductList;
 }
 
 export function MetaTags({ productList }: MetaTagsProps) {
-   const appContext = useAppContext();
-   const currentRefinements = useCurrentRefinements();
+   const itemTypeOverrides = useItemTypeOverrides(productList);
+   const metaTitle = useMetaTitle(productList);
+   const metaDescription =
+      itemTypeOverrides?.metaDescription ??
+      productList.metaDescription ??
+      itemTypeOverrides?.metaDescription ??
```

Comment:
I think you were going for this:
```suggestion
      itemTypeOverrides?.description ??
```